react/tic-tac-toe/src/app/screens/Login/components/LoginForm/styles.module.scss

```diff
@@ -1,10 +1,32 @@
 @import '../../../../../scss/colors';
 
-.button {
-  align-self: flex-start;
-}
-
 .error-msg {
   color: $red-04;
   display: block;
-}
\ No newline at end of file
+}
+
+.form-container {
+  border-radius: 5px;
+  box-shadow: 2px 2px 13px 1px $black-07;
+  height: 200px;
+  margin: auto;
+  width: 400px;
+  
+  .submit {
+    align-self: center;
+    background: $dust-gray;
+    border: 1px solid $dust-gray;
+    border-radius: 5px;
+    color: white;
+    height: 30px;
+    line-height: 30px;
+    width: 100px;
+  }
+
+  .title {
+    align-self: center;
+    color: $black;
+    font-size: 22px;
+    padding: 15px 0;
+  }
+}
```

Comment:
Same, be name specific. 

https://medium.com/wolox-driving-innovation/what-margin-padding-taught-me-after-50-projects-6878b53c903f

There you can find more about it. That's the way we handle it in wolox